README.md

```diff
@@ -5,3 +5,53 @@ A general template repository for default settings when creating new repositorie
 This repository uses the Apache-2.0 license. `COPYRIGHT.txt` contains a current copyright statement which should be included at the top of all files.
 
 When creating a new repository you [can use this repository as a template](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template), to automate the creation of the correct license and COPYRIGHT statement.
+
+## COPYRIGHT Header
+
+Best practice suggests adding a copyright statement at the top of every source code file, or text file where it is possible to add a copyright statement without interfering with the purpose of the file. The reasoning is if a file is separated from the repository in which it resides then it may not be possible to ascertain it's licensing, which may hamper re-use.
+
+Making this as short and concise as possible reduces the overhead in including such a copyright statement. To that end using [SPDX identifiers](https://spdx.dev/ids/) is simple, efficient, portable and machine-readable.
+
+### Examples
+
+An example, short, copyright statement is reproduced below, as it might appear in different coding languages. Copy and add to files as appropriate: 
+
+#### plaintext
+```text
+*********************************************************************************************
+* Copyright 2022 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details. *
+* SPDX-License-Identifier: Apache-2.0                                                       *
+*********************************************************************************************
+```
```

Comment:
Why is it the only one enclosed between 2 lines of symbols? I'm no big fan of the lines of symbols...

Reply:
No big reason, but all the others are in code, so it is clearly "comment", there is no such thing in a plaintext file, so it is one of the ways of showing that this is not part of the body of the file. Happy to remove it, or replace that with something else that does that job.

Examples here suggest that perhaps a line at the bottom would be better

https://www.termsfeed.com/blog/sample-copyright-notices/

e.g.

© 2022 ACCESS-NRI and contributors. See the top-level COPYRIGHT file for details. SPDX-License-Identifier: Apache-2.0

Reply:
I've changed to the example above (sort of).